hello-hwasan/README.md

```diff
@@ -0,0 +1,57 @@
+Hello-hwasan
+=========
+hello-hwasan is an Android sample that shows how to build an application with [HWAddress Sanitizer](https://developer.android.com/ndk/guides/hwasan).
+
+Introduction
+------------
+
+This very simple app is built with HWAddressSanitizer (HWASan) and provides several ways to trigger a native crash to see the memory debugging tool in action.
+
+This app needs a HWASan-build Android image to run. Refer to Android documentation on how to [acquire](https://developer.android.com/ndk/guides/hwasan) or [build](https://source.android.com/devices/tech/debug/hwasan) one.
+
+Pre-requisites
+--------------
+- Android Studio 2.2+ with [NDK](https://developer.android.com/ndk/) bundle.
+
+Getting Started
+---------------
+1. [Download Android Studio](http://developer.android.com/sdk/index.html)
+1. Launch Android Studio.
+1. Open the sample directory.
+1. Open *File/Project Structure...*
+  - Click *Download* or *Select NDK location*.
+1. Click *Tools/Android/Sync Project with Gradle Files*.
+1. Click *Run/Run 'app'*.
+
+Screenshots
+-----------
+![screenshot](screenshot.png)
+
+Support
+-------
+If you've found an error in these samples, please [file an issue](https://github.com/googlesamples/android-ndk/issues/new).
```

Comment:
This URL will redirect here, but we should probably try to use android/ndk-samples rather than googlesamples/android-ndk when we're adding new samples (although @ggfan is the owner here, so it's up to him)